package/MDAnalysis/analysis/rms.py

```diff
@@ -484,6 +486,63 @@ def save(self, filename=None):
         return filename
 
 
+def rmsf(atomgroup, start=0, stop=-1, step=1, progout=None):
```

Comment:
Although I understand that classes are more expandable, I don't think there's much to be gained from implementing a piece of code that obtains RMSFs from a given set of atoms as a class, at least not for a minimal implementation. In any case, we already have an ``rmsd`` function that is less full-featured than the ``RMSD`` class, and I don't think this is a bad thing. In fact, I think the ``rmsd`` function should be made even simpler than its current implementation (or at least given a keyword to *not* perform fitting). Is the ``rmsd`` function marked for deprecation?

I'm happy to build an ``RMSF`` class later, but this would take much longer to write. Would we prefer having an ``rmsf`` function now with an implementation and API that is unlikely to change or an ``RMSF`` class that could be very much in flux for some time?